### PR TITLE
IMPROVE INSTALLER - LANDrop

### PR DESCRIPTION
**Pull request by CoolPlayLin**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/108819)

### CODE REVIEW

**Comment by CoolPlayLin** [comment]
@stephengillie

**Comment by CoolPlayLin** [comment]
@ImJoakim

**Comment by stephengillie** [comment]
Manual Validation ended with: 
> 2023-06-05 11:28:11.445 [FAIL] Installer failed security check. Url: https://github.com/CoolPlayLin/LANDropSetup/releases/download/0.4.0/LANDrop.Setup.ia32.User.exe Result: 0x80004005

**Comment by CoolPlayLin** [comment]
What the security issue is?

**Comment by CoolPlayLin** [comment]
@stephengillie

**Comment by CoolPlayLin** [comment]
image
The Installer was blocked by Windows Defender

**Comment by CoolPlayLin** [comment]
@ItzLevvie

**Comment by CoolPlayLin** [comment]
What should I do to solve that?

**Comment by CoolPlayLin** [comment]
@ItzLevvie @stephengillie

File: manifests/c/CoolPlayLin/LANDrop/0.4.0/CoolPlayLin.LANDrop.installer.yaml [hunk 1]
```diff
@@ -3,10 +3,17 @@
...
 Installers:
 - Architecture: x64
-  InstallerType: inno
+  Scope: machine
   InstallerUrl: https://github.com/CoolPlayLin/LANDropSetup/releases/download/0.4.0/LANDrop.Setup.ia32.System.exe
   InstallerSha256: F8594706B60138D4202A24C46BDC9E0B5F2389623870A8BC7F823622311BCEED
+- Architecture: x64
+  Scope: user
+  InstallerUrl: https://github.com/CoolPlayLin/LANDropSetup/releases/download/0.4.0/LANDrop.Setup.ia32.User.exe
+  InstallerSha256: D364EB7647835A586BD245F5286911E7074F1BBFF4B007891C324B310F9671E2
```
Comment (ItzLevvie) [inline manifests/c/CoolPlayLin/LANDrop/0.4.0/CoolPlayLin.LANDrop.installer.yaml:17]: I'm seeing this:
image
image
image

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by CoolPlayLin** [comment]
I cannot fix that

**Review by ItzLevvie — APPROVED** [review]
Seems like it's fine now:
image
image

**Comment by stephengillie** [comment]
I'm not able to determine the cause for the `Validation-Executable-Error` label - these install normally in Windows 11 VMs.

**Comment by CoolPlayLin** [comment]
What can I do to fix that?

**Comment by CoolPlayLin** [comment]
No one?

**Comment by stephengillie** [comment]
> No one?

Apologies for the delay. This package installs normally, but requires `Microsoft.VCRedist.2015+.x64` to run:
image

File: manifests/c/CoolPlayLin/LANDrop/0.4.0/CoolPlayLin.LANDrop.installer.yaml [hunk 2]
```diff
@@ -3,10 +3,17 @@
 
 PackageIdentifier: CoolPlayLin.LANDrop
 PackageVersion: 0.4.0
+Platform:
+  - Windows.Desktop
+InstallerType: inno
```
Comment (stephengillie) [inline manifests/c/CoolPlayLin/LANDrop/0.4.0/CoolPlayLin.LANDrop.installer.yaml:8]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
InstallerType: inno
```

**Comment by CoolPlayLin** [comment]
> @wingetbot waivers Add Validation-Executable-Error

@stephengillie #111148 needs to review.

**Comment by CoolPlayLin** [comment]
This microsoft/winget-cli#163 has been resolved?

**Review by stephengillie — APPROVED** [review]

**Pull request merged**